ebos/equil/initstateequil.hh

```diff
@@ -544,7 +544,7 @@ private:
     /// \param[in] pcow O/W capillary pressure value (Po - Pw).
     ///
     /// \return Water saturation value.
-    double applySwatInit(const double pcow);
+    std::tuple<double, bool> applySwatInit(const double pcow);
```

Comment:
Why `tuple<>` instead of `pair<>`?

Reply:
I saw some other code using `tuple` so I used it as well. Honestly didn't know about `pair`, but that's more logical! I have changed it here and in the opm-common PR.